plugins/rbac-backend/src/service/role-manager.ts

```diff
@@ -240,7 +240,7 @@ export class BackstageRoleManager implements RoleManager {
    * domain is a prefix to the roles.
    */
   async getRoles(_name: string, ..._domain: string[]): Promise<string[]> {
-    throw new Error('Method "getRoles" not implemented.');
+    return Promise.resolve([...this.allRoles.keys()]);
```

Comment:
So, this is returning all roles within Casbin, but the comment above mentions that it returns the roles that the name inherits. Should we instead filter through `allRoles` and return the roles that a user is associated with using `hasLink`? Or will returning `allRoles` be fine for other inheritance cases?

I'm seeking clarification since I haven't seen into many inheritance scenarios come up yet, making me wonder about what potential issues may arise from returning all roles through `getRoles`.

Reply:
@PatAKnight , thanks. Really great catch. Awesome!